src/groups/mqb/mqbblp/mqbblp_rootqueueengine.h

```diff
@@ -140,6 +140,9 @@ class RootQueueEngine BSLS_KEYWORD_FINAL : public mqbi::QueueEngine {
     // Throttler for when reject messages
     // are dumped into temp files.
 
+    QueueEngineUtil_AppsDeliveryContext d_context;
```

Comment:
just a suggestion, since this is a data member now, a bit more descriptive name to disambiguate from other (future) contexts